wp/resources/sample_binaries/user_func_spec/sub_spec_5/run_wp_1.sh

```diff
@@ -0,0 +1,13 @@
+# Should return SAT: we provide a spec only for g, which means f can still return 0x67 and we can trip the assert
+
+run () {
+  bap wp \
+    --func=main \
+    --precond="(assert (= (bvand RDI #xFFFFFFFF00000000) #x0000000000000000))" \
```

Comment:
Do you need these?

Reply:
Nope! I copied from other tests. Could be removed from this one and others

Reply:
Probably good not to have useless annotations in tests.

Reply:
K they are removed